Art/ChuxMg/style.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  background-color: #16d277;
+  display: flex;
+  width: 1000px;
+}
+
+
+.heading {
+  background-color: #15202B;
+  color: #8B98A5;
+  font-weight: bold;
+  font-family: 'Trebuchet MS';
+  font-size: 30px;
+  padding: 0px 30px;
+}
+
+.text-slide {
+  --c: #fff;
```

Comment:
# Invalid CSS rules
- Please correct or remove the code ( most likely the color rule which is already stated bellow )